### ISSUE TITLE
Remove FOF library

### ISSUE THREAD

**Issue by johanjanssens** [joomladevelopers/joomla-spark#77]
Remove the Framework of a Framework library

See: https://github.com/joomlatools/joomlatools-platform/issues/93

**Comment by n2h** [comment]
I have tried to complete this issue but got a big problem: com_postinstall is totally based on FOF so we'll need to create a new com_postinstall if we want to remove FOF:

```// Dispatch the component.
FOFDispatcher::getTmpInstance('com_postinstall')->dispatch();```

**Comment by johanjanssens** [comment]
Thanks @n2h. This one will need a bit more research first indeed, maybe we can start by making a list of issues we see when we try to remove this? Can create a wiki page about it and then from there decide how to move forward?

**Comment by n2h** [comment]
I have rechecked and found that FOF is required in "com_postinstall" and plugin "twofactorauth".
As we'll remove plugin "twofactorauth" by issue 78, so rewrite "com_postinstall" is the only issue needed.

**Comment by johanjanssens** [comment]
Yes, that's correct, the twofactorauth is very coupled in various aspects of the core. This needs to be removed and can in a later step by rebuild. The postinstall component is something we going to need to refactor might need to try and wrap that as part of the installer component, as both are very related.
